create-escrow.js

```diff
@@ -35,17 +16,22 @@ async function createEscrow(env, manifestUrl, manifestHash) {
   const signer = new ethers.Wallet(privateKey, provider);
 
   const escrowClient = await EscrowClient.build(signer);
+  
   const stakingClient = await StakingClient.build(signer);
 
-  const manifest = await fetchManifest(manifestUrl);
-
-  const tokenAddress = getTokenAddress(chainId, manifest.fund_token);
-  const tokenContract = new ethers.Contract(tokenAddress, ERC20ABI, signer);
+  const tokenAddress = getTokenAddress(chainId, env.REWARD_TOKEN);
+  const tokenContract = new ethers.Contract(
+          tokenAddress,
+          ERC20ABI,
+          signer
+        );
+  const tokenDecimals = await tokenContract.decimals();
+  const _tokenDecimals = Number(tokenDecimals) || 18;
 
   const fundAmount = ethers.parseUnits(
-    env.REWARD_AMOUNT,
-    await tokenContract.decimals()
-  );
+          env.REWARD_AMOUNT.toString(),
+          _tokenDecimals
+        );
```

Comment:
Avoid 2 variables with the same name
```suggestion
  const tokenDecimals = Number(await tokenContract.decimals()) || 18;

  const fundAmount = ethers.parseUnits(
          env.REWARD_AMOUNT.toString(),
          tokenDecimals
        );
```